types/frappe-gantt/frappe-gantt-tests.ts

```diff
@@ -36,4 +36,5 @@ new Gantt('#gantt', tasks, {
         </div>
       `;
     },
+    language: "es",
```

Comment:
That's fine, but please change to single quotes (to match the rest of the tests code).

Reply:
done